### PR TITLE
REP-2478 Added support for the new Repose Phone Home service.

### PR DESCRIPTION
**Pull request by wdschei**
We probably want to extract the `repose-phone-home.erb` from this project into the `repose-phone-home-service` and hard-code the templated `USER` and `GROUP`.

This could also be done in the `repose_phone_home/manifests/init.pp` which would allow the `daemon_user` and `daemon_group` variables to be removed from the `common.eyaml`.

### CODE REVIEW

File: myModules/repose_phone_home/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+`scp -p  ~/keys/eyaml/public_key.pkcs7.pem                           root@et.openrepose.org.bast:/etc/puppet/ssl/`
+`scp -p  ~/keys/eyaml/private_key.pkcs7.pem                          root@et.openrepose.org.bast:/etc/puppet/ssl/`
+`scp -p  <REPOSE_INFRASTRUCTURE_NG_DIR>/hiera.yaml                   root@et.openrepose.org.bast:/etc/puppet/`
+`scp -rp <REPOSE_INFRASTRUCTURE_NG_DIR>/hieradata/                   root@et.openrepose.org.bast:/etc/puppet/`
+`scp -rp <REPOSE_INFRASTRUCTURE_NG_DIR>/myModules/repose_phone_home/ root@et.openrepose.org.bast:/etc/puppet/modules/`
+
+## Remote test server
+
+### Install the module's dependencies
+`puppet module install puppetlabs-mongodb`
```
Comment (dkowis) [inline myModules/repose_phone_home/tests/README.md]: If you use the `Puppetfile` and `librarian-puppet` dependencies are automatically handled for you.

Comment (dkowis) [inline myModules/repose_phone_home/tests/README.md]: Oh to test it. Yeah, there are ways within puppet as to how to test modules, but it's super not trivial :(

Maybe you could use vagrant and http://serverspec.org/ ?

File: myModules/repose_phone_home/manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  $mongo_port     = 27017,
+  $mongo_username = undef,
+  $mongo_password = undef,
+  $mongo_dbname   = undef,
+) {
+
+#  class {'::mongodb::globals':
+#    manage_package_repo => true,
+#  } ->
+  class {'::mongodb::server':
```
Comment (adrianjgeorge) [inline myModules/repose_phone_home/manifests/init.pp]: will this install mongo? will it do the right version?

File: myModules/repose_phone_home/manifests/init.pp [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  package{ "openjdk-6-jre-headless":
+    ensure => absent,
+  }
+
+  package{ "openjdk-7-jre-headless":
+    ensure => present,
+  }
+
+  package{ "repose-phone-home":
```
Comment (adrianjgeorge) [inline myModules/repose_phone_home/manifests/init.pp]: also need our package repository

**Pull request merged**